Makefile

```
@@ -64,26 +64,28 @@ pull: ## update the Docker image used by "make shell"
 	docker pull edxops/edxapp:latest
 
 pre-requirements: ## install Python requirements for running pip-tools
+	pip install -qr requirements/pip.txt
 	pip install -qr requirements/edx/pip-tools.txt
 
 local-requirements:
 # 	edx-platform installs some Python projects from within the edx-platform repo itself.
 	pip install -e .
 
-dev-requirements: local-requirements
-	pip install -qr requirements/edx/development.txt
-
-base-requirements: local-requirements
-	pip install -qr requirements/edx/base.txt
-
-test-requirements: local-requirements
-	pip install --exists-action='w' -qr requirements/edx/testing.txt
-
-requirements: pre-requirements ## install development environment requirements
+dev-requirements: pre-requirements ## install development environment requirements
 	@# The "$(wildcard..)" is to include private.txt if it exists, and make no mention
 	@# of it if it does not.  Shell wildcarding can't do that with default options.
 	pip-sync -q requirements/edx/development.txt $(wildcard requirements/edx/private.txt)
-	pip install -e .
+	make local-requirements
+
+base-requirements: pre-requirements
+	pip-sync requirements/edx/base.txt
+	make local-requirements
+
+test-requirements: pre-requirements
+	pip-sync --pip-args="--exists-action=w" requirements/edx/testing.txt
+	make local-requirements
+
+requirements: dev-requirements
```

tiniest of nits: adding this comment back could make it a little clearer that this target installs dev requirements, not base requirements.

```suggestion
requirements: dev-requirements ## install development environment requirements
```

---

Oh, yeah. :-)

---

and maybe remove it from `dev-requirements`, which is kind of self-explanatory

---

👍🏻